lib/arel/nodes/unary_operation.rb

```diff
@@ -0,0 +1,25 @@
+module Arel
+  module Nodes
+
```

Comment:
Just leave it here.

On Wed, Jan 6, 2016, 11:53 Shahbaz Javeed <notifications@github.com> wrote:

> In lib/arel/nodes/unary_operation.rb
> <https://github.com/rails/arel/pull/408#discussion_r48958258>:
>
> > @@ -0,0 +1,25 @@
> > +module Arel
> > +  module Nodes
> > +
>
> @rafaelfranca <https://github.com/rafaelfranca> this blank line follows
> the pattern in infix_operation.rb. Did you want me to remove it from there
> as well or just leave it here?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rails/arel/pull/408/files#r48958258>.
>